components/vocabularies/omim/api/src/main/java/org/phenotips/vocabulary/internal/solr/MendelianInheritanceInMan.java

```diff
@@ -155,8 +248,8 @@ private SolrQuery addGlobalQueryParameters(SolrQuery query)
     private SolrQuery addFieldQueryParameters(SolrQuery query)
     {
         query.set(DisMaxParams.PF, "name^40 nameSpell^70 synonym^15 synonymSpell^25 text^3 textSpell^5");
-        query.set(DisMaxParams.QF,
-            "name^10 nameSpell^18 nameStub^5 synonym^6 synonymSpell^10 synonymStub^3 text^1 textSpell^2 textStub^0.5");
+        query.set(DisMaxParams.QF, "name^10 short_name^5 included_name^5 nameSpell^18 nameStub^5 "
```

Comment:
The `Stub` and `Spell` versions of `short_name` and `included_name` should also be included.